syntax = "proto3";

message gate_addr_notify {
	string addr = 1;
}